command/apply_test.go

```diff
@@ -2130,26 +2130,30 @@ func TestApply_jsonGoldenReference(t *testing.T) {
 	}
 
 	// Compare the rest of the lines against the golden reference
-	for i := range gotLines[1:] {
+	var gotLineMaps []map[string]interface{}
+	for i, line := range gotLines {
```

Comment:
If I'm reading it correctly, I believe this change is going to result in a test failure when the version number increments. The test was intentionally _not_ comparing the initial `"type": "version"` message, so that we don't have to touch the golden reference file every time we make a release.

Reply:
Oh, thanks for catching this! I had intended to make this `range gotLines[1:]` as before but I seem to have got my wires crossed when I was reorganizing this.

My goal here was just to make the output easier to scan when something changes, as happened here, because with it as before I found it quite hard to understand the difference between the golden example and the current output. But I didn't intend to change what this is testing, so I'll fix this up to make sure it's skipping out the version item as before.